### ISSUE TITLE
Menu not fully accessible in mobile webbrowser

### ISSUE THREAD

**Issue by XavierNV** [RickStrahl/HelpBuilderReleases#67]
If you display the help on a mobile phone or on a webbrowser control using a mobile "user-agent", when you click on then menu button to show it you cannot access the end of the treeview menu if you open it. A scrollbar is lacking... (as in Helpndoc or Help+Manual systems...).
Thanks to consider this issue...

**Comment by RickStrahl** [comment]
Yup that's definitely a bug. What's supposed to happen is that the horizontal scrolling should be off but vertical scrolling should most definitely work.

**Comment by RickStrahl** [comment]
As a quick work around you can fix the CSS in`wwhelp.css` (or **Edit Topic Templates -> Edit CSS Stylesheet** as follows:

```css
@media(max-width: 767px) {
    .sidebar-left {
        width: 0;
        x-position: absolute;        
        z-index: 100;
    }  
}
```

Basically remove the `position:absolute` which improperly throws the sizing off. This will make mobile displays work although for mobile phones the displaced content will be a bit ugly.

We'll look into a better solution.

**Comment by XavierNV** [comment]
Thanks a lot for your answer Rick. I will have a try…
Xavier


De : Rick Strahl<mailto:notifications@github.com>
Envoyé le :mercredi 21 février 2018 05:09
À : RickStrahl/HelpBuilderReleases<mailto:HelpBuilderReleases@noreply.github.com>
Cc : XavierNV<mailto:xni@live.fr>; Author<mailto:author@noreply.github.com>
Objet :Re: [RickStrahl/HelpBuilderReleases] Menu not fully accessible in mobile webbrowser (#67)


As a quick work around you can fix the CSS inwwhelp.css (or Edit Topic Templates -> Edit CSS Stylesheet as follows:

@media(max-width: 767px) {

    .sidebar-left {

        width: 0;

        x-position: absolute;

        z-index: 100;

    }

}

Basically remove the position:absolute which improperly throws the sizing off. This will make mobile displays work although for mobile phones the displaced content will be a bit ugly.

We'll look into a better solution.
